pkgs/development/python-modules/pycm/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, isPy3k, numpy, coverage, pytest }:
+
+buildPythonPackage rec {
+  pname = "pycm";
+  version = "2.5";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner  = "sepandhaghighi";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0zfv20hd7zq95sflsivjk47b0sm7q76w7fv2i2mafn83ficzx0p0";
+  };
+
+  # remove a trivial dependency on the author's `art` Python ASCII art library
+  patchPhase = ''
+    rm pycm/__main__.py
+    substituteInPlace setup.py --replace '=get_requires()' '=[]'
+  '';
+
+  checkInputs = [ pytest ];
+  buildInputs = [ coverage ];
```

Comment:
It's more likely this is for `checkInputs`

Reply:
Doesn't seem needed actually...